### PR TITLE
Toggle Price Feeds list to be collapsible

### PR DESCRIPTION
**Pull request by sirromdev**

### CODE REVIEW

**Comment by jparklev** [comment]
also note that, b/c of [effect cleanup](https://reactjs.org/docs/hooks-reference.html#cleaning-up-an-effect), it's usually good practice to avoid implicit returns in the `useEffect` callback

File: src/components/SidebarFeeds.js [hunk 1]
```diff
@@ -1,35 +1,56 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```
Comment (levity) [inline src/components/SidebarFeeds.js]: make sure to remove unused imports (do you have a linter set up to flag these in your IDE?)

**Comment by levity** [comment]
Hmm, stand by. I recall seeing a mockup in Figma that used actual "Show more" and "Show less" links at the bottom of the price feeds list rather than a flipping caret as you've implemented it here. But that seems to have gone missing from the Figma doc. /cc @gitdk @mikemckain

**Comment by mikemckain** [comment]
Yeah, thanks Lawrence. [Here are the most up-to-date price feeds.](https://www.figma.com/file/WD1TVy5hFtVkWgd7hvhbzFIe/Components?node-id=1810%3A4) 

They hadn't been changed in the MCD CDP project. I've added them to the collateral page.

**Comment by levity** [comment]
Ok cool, thanks Michael. @sirromdev please implement to match the mock-up.

File: src/components/SidebarFeeds.js (hunk 1 above)
Comment (sirromdev) [reply src/components/SidebarFeeds.js]: done. I do, but it's not as intrusive with these errors a it could be. I'll check on configuring it better

**Comment by sirromdev** [comment]
I've updated those changes @levity

**Pull request merged**